src/app/components/shell/header/header.scss

```diff
@@ -44,182 +74,583 @@
+            opacity: 1;
+            transition: opacity 0.5s;
+
+            @include small {
+              @at-root .active#{&} {
+                opacity: 0;
+                transition: opacity 0.5s;
+              }
+            }
+          }
+          svg #grey {
+            @include small {
+              @at-root .active#{&} {
+                opacity: 1;
+              }
+            }
+          }
+
+          .logo-quote {
             display: inline-block;
+            transform: scale3d(0,0,0);
+            color: #5E6062;
+            font-size: 1.4rem;
+            position: absolute;
+            vertical-align: text-top;
+            margin-left: 20px;
+            top: 1px;
+
+            @include large {
+              position: relative;
+              transform: scale3d(1,1,1);
+              transition: transform 0.3s;
+            }
+          }
+      } //end logo-wrapper
+
+      .nav-menu {
+        padding: 0 0 0 8rem;
+        list-style: none;
+        flex: 1 100%;
+        order: 3;
+        margin: 1.5rem 0 0;
+        //transform: translate3d(0, 0, 0);
+
+        @include small {
+          @at-root .active.open#{&} {
+            transform:translate3d(-1000px, 0, 0);
+            transition: transform 0.2s;
+          }
         }
-    }
 
-    .logo {
-        display: inline-block;
-        margin-right: 2.5rem;
+        @include medium-up {
+          order: 2;
+          flex: 0 0 auto;
+          display: flex;
+          margin-top: 0px;
+          margin-left: auto;
+          padding: 0;
+          transform: translate3d(0, 0, 0);
+          transition: 0.3s transform cubic-bezier(.3,.73,.3,.74);
+
+          @at-root .sticky#{&} {
+            opacity: 1;
+            transition: 0.3s transform cubic-bezier(.3,.73,.3,.74), opacity 0.5s;
+          }
+          @at-root .sticky.active#{&} {
+            transform: translate3d(0, 70px, 0) scale3d(0,0,0);
+            transition: 0.3s transform cubic-bezier(.3,.73,.3,.74), opacity 0.1s;
+            opacity: 0;
+          }
+        }
 
-        .logo-color {
-            display: inline-block;
+        .nav-menu-item {
+          margin: 0 0 1rem;
 
-            @at-root .transparent#{&} {
-                display: none;
+          @include medium-up {
+            margin: 0px;
+            //padding: 1rem 0;
+          }
+
+          &.parent {
+            @include medium-up {
+              position: relative;
             }
-        }
+            &.open {
+              @include small {
+                @at-root .active.open#{&} {
+                  position: absolute;
+                  top: 0px;
+                  transform: translate3d(1000px,0,0);
+                  transition: transform;
+                }
+              }
+              .back {
+                @include small {
+                  @at-root .active.open#{&} {
+                    width: 27px;
+                    height: 26px;
+                    text-indent: -9999px;
```

Comment:
This is an old hack to hide something off screen, but it's still visible to screen readers.  If something should not be visible, can it be down using `display: none` or some other more standard CSS that would also properly convery intent to screen readers?